src/scripts/Game.ts

```diff
@@ -84,6 +84,7 @@ class Game {
         this.keyItems.initialize();
         this.oakItems.initialize();
         this.farming.initialize();
+        this.load();
 
         // TODO refactor to proper initialization methods
         Battle.generateNewEnemy();
```

Comment:
This was the line causing the problem for the log book.

Reply:
That's weird, do you have any idea why?

Reply:
It’s because the party hasn’t loaded yet, so every Pokémon is counted as uncaught.